.gitignore

```diff
@@ -19,5 +19,21 @@
 *.tar.gz
 *.rar
 
+# OS files
+.DS_Store
+
 # virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
 hs_err_pid*
+
+### IntelliJ IDEA ###
+.idea
+*.iws
+*.iml
+*.ipr
+
+target/
+.classpath
+.project
+.settings/
+.vscode/
+.factorypath
```

Comment:
When creating other Spring boot apps, I was told to follow this gitignore file example:

`#Build artifacts
target/
*.jar
*.zip

# Eclipse files
.classpath
.project
.settings
bin/

# IntelliJ files
.idea/
*.iml

# Mac files
.DS_Store
._*
`